### ISSUE TITLE
move away from Sqlite into Postgre

### ISSUE THREAD

**Issue by atherdon** [atherdon/dash-back#105]
I think we can use Postgre from Heroku, as they are the major Paas for it.

https://www.heroku.com/postgres

**Comment by kolserdav** [comment]
Yes, it possible. Simple run database and set up database url in dash-back, and also postges settings for prisma.
